pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -20,13 +20,25 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"k8s.io/kubernetes/pkg/apis/componentconfig"
 	apitest "k8s.io/kubernetes/pkg/kubelet/api/testing"
+	containertest "k8s.io/kubernetes/pkg/kubelet/container/testing"
+	"k8s.io/kubernetes/pkg/kubelet/network"
+	nettest "k8s.io/kubernetes/pkg/kubelet/network/testing"
 )
 
-func createTestFakeRuntimeManager() (*apitest.FakeRuntimeService, *apitest.FakeImageService, *kubeGenericRuntimeManager, error) {
+func createTestFakeRuntimeManager() (*apitest.FakeRuntimeService, *apitest.FakeImageService, *KubeGenericRuntimeManager, error) {
```

Comment:
In general, we create `Fakes` so that other packages can test with this single component (`KubeGenericRuntimeManager`) mocked out. In this case, you are testing the component itself in the package. Suggest not reuse the `NewFakeKubeRuntimeManager` here.